app/objects/secondclass/c_host.py

```diff
@@ -58,3 +74,6 @@ def __init__(self, ip, hostname=None, ports=None, cves=None, software=None, os=N
         self.denied_abilities = denied_abilities or []
         self.access = access or 0
         self.access_prob = access_prob or 1.0
+
+    def __repr__(self):
```

Comment:
This representation is useful for debugging but may be depreciated before `pathfinder-v2` is merged in to `main`.